src/libstd/collections/hash/set.rs

```diff
@@ -757,6 +757,16 @@ impl<T, S> Eq for HashSet<T, S>
 {
 }
 
+#[unstable(feature = "hashmap_hash", issue = "0")]
```

Comment:
impls are insta-stable

Reply:
hmm yes, true... maybe it should be `#[stable(feature = "hashmap_hash", issue = "1.26.0")]`?